robot_vision.py

```diff
@@ -0,0 +1,59 @@
+"""
+The main file to call that takes vision tracking information and sends to
+a network table.
+Use `java -jar support/OutlineViewer-1.0.1.jar` in terminal to create a local
+server, once you have everything installed (see README)
+"""
+from lib import config, tables, target_tracker, color_mask, util
+
+def run(cfg):
+    channel = cfg.get('channel')
+    server = cfg.get('networktables')
+    lu = cfg.get("color", "yellow")
+    lower, upper = color_mask.unpack_range(lu)
+    camera, width, height = util.get_video(channel)
+    DEBUG = cfg.get_default("debug", False)
```

Comment:
I like this `DEBUG` option, however, since it isn't a global variable, it could be lowercased.